tests/test_mariadb.py

```diff
@@ -224,3 +211,30 @@ def wait_for_server():
     assert rows and len(rows) == 2
     _, num, data = rows[-1].split()
     assert num == "100" and data == "abcdef"
+
+
+def test_mariadb_healthcheck_innodb_initialized(auto_container_per_test):
+    """
+    Test if InnoDB (storage engine) has completed initializing.
+
+    See: <https://mariadb.com/kb/en/using-healthcheck-sh/#-innodb_initialized>
+    """
+    conn = auto_container_per_test.connection
+
+    _wait_for_server(conn)
+
+    conn.check_output("healthcheck.sh --su-mysql --innodb_initialized")
+
+
+def test_mariadb_healthcheck_galera_cluster_disabled(auto_container_per_test):
+    """
+    Ensure that Galera Cluster (multi-primary cluster) is disabled (experimental feature),
+    i.e. :command:`healthcheck.sh --su-mysql --galera_online` fails.
+
+    See: <https://mariadb.com/kb/en/using-healthcheck-sh/#-galera_online>
```

Comment:
Not a proper rst link:
```suggestion
    See: `<https://mariadb.com/kb/en/using-healthcheck-sh/#-galera_online>`_
```